diff --git a/source/header.php b/source/header.php
index 9436049..21f7e32 100644
--- a/source/header.php
+++ b/source/header.php
@@ -19,6 +19,17 @@
 </head>
 
 <body <?php body_class(); ?>>
+
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-38673415-2', 'auto');
+  ga('send', 'pageview');
+
+</script>
 	<?php if ( get_header_image() ) : ?>
 		<div id="header-image"></div>
 	<?php endif; // End header image check. ?>
